### ISSUE TITLE
vs-code does not work with latest nightly gauge

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/gauge-vscode#388]
**Excepted Result:**
 All the vs code feature code should working

**Actual Result:**
vs-code feature does not work. 

**Steps to reproduce:**
1. Install gauge nightly in a fresh system
2. run `gauge init java`
3. open project in vs-code

```
version:
auge version: 1.0.6.nightly-2019-07-09
Commit Hash: df8d663
```

Plugins
-------
html-report (4.0.8)
java (0.7.3.nightly-2019-05-24)

**Comment by Apoorva-GA** [comment]
Fix should be available in gauge nightly >= 10/7/19

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the version ```1.0.6.nightly-2019-07-10```
